src/schema/schema.js

```diff
@@ -0,0 +1,60 @@
+const { buildSchema } = require('graphql');
+
+const schema = buildSchema(`
```

Comment:
@michaelmurray6298 You'll see below I make several suggestions for renaming fields. You will need to update the rest of the files as appropriate. Also, I notice some of the field names start with a capital letter and others with a lowercase, please choose one and standardize it across the schema, thanks!